docs/moderator-handbook.md

```diff
@@ -187,11 +193,15 @@ In exceptional circumstances, it may be appropriate for a moderator to fork a di
 
 Again, if you have any problems or questions, make a post with your actions in the `"Staff"` category, and tag another moderator if you want them to review your moderating actions.
 
+### Dealing with Posted Solutions
+
+When a user posts a solution to any of the questions, remove it and either, kindly let them know that solutions are not permitted on the forum in your own words or, use the **Solution Instead of Help** "Canned reply".
+
```

Comment:
Personally, I would not add this to a moderator handbook.

It is not spammy to post a solution (once). We will just never encourage the sharing/posting of solutions, because it rarely aids anyone's learning.

Essentially, it is not a "must remove solution" rule; it is a "encourage proactive learning" rule.

Reply:
yes true.  I think we can rewrite to:

When a user responds to another user's post by posting a solution, remove the solution and suggest that tips/hints are a better way to help others learn (alt. use the **Solution Instead of Help** "Canned reply").

Reply:
> Essentially, it is not a "must remove solution" rule; it is a "encourage proactive learning" rule.

The practise lately, AFAIK, has been to remove/edit out a solution in a post if that solution was provided in response to a camper's request for help, but that a solution being shared for the sake of discussing a solution is fine.